war/src/main/js/add-item.js

```diff
@@ -0,0 +1,289 @@
+// Initialize all modules by requiring them. Also makes sure they get bundled (see gulpfile.js).
+var $ = require('jquery-detached').getJQuery();
+
+var getItems = function(root){
+  var d = $.Deferred();
+  $.get(root+'/categories?depth=3').done(
+      function(data){
+        d.resolve(data);
+      }
+  );
+  return d.promise();
+}; 
+
+var jRoot = $('head').attr('data-rooturl');
+
+$.when(getItems(jRoot)).done(function(data){
+  $(function() {
+    
+    //////////////////////////////
+    // helpful reference DOM
+
+    var defaultMinToShow = 2;
+    var $root = $('#main-panel');
+    var $form = $root.find('form[name="createItem"]').addClass('jenkins-config new-view');
+    var $newView = $('<div class="new-view" />')
+      .attr('name','createItem')
+      .attr('action','craetItem')
```

Comment:
I think @kzantow told something before, anyway, `.attr('action','creatItem')` instead of `.attr('action','craetItem')`